mdsuite/file_io/extxyz_files.py

```diff
@@ -0,0 +1,305 @@
+"""
+MDSuite: A Zincwarecode package.
+
+License
+-------
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License v2.0 which accompanies this distribution, and is
+available at https://www.eclipse.org/legal/epl-v20.html
+
+SPDX-License-Identifier: EPL-2.0
+
+Copyright Contributors to the Zincwarecode Project.
+
+Contact Information
+-------------------
+email: zincwarecode@gmail.com
+github: https://github.com/zincware
+web: https://zincwarecode.com/
+
+Citation
+--------
+If you use this module please cite us with:
+
+Summary
+-------
+"""
+import logging
+import typing
+from typing import Union, List, Dict, Tuple
+import numpy as np
+import pathlib
+import copy
+import tqdm
+
+import mdsuite.file_io.file_read
+import mdsuite.database.simulation_database
+from mdsuite.utils.meta_functions import get_dimensionality
+
+log = logging.getLogger(__name__)
+
+var_names = {
+    "Positions": "pos",
+    "Velocities": "vel",
+    "Forces": "force",
+    "Stress": "stress",
+    "PE": "energies",
+    "Time": "time",
+    "Lattice": "Lattice",
+    "Momenta": "momenta",
+}
```

Comment:
I think this could also be a dataclass? 
Even if not used as such and converted via `dataclasses.asdict` it looks cleaner

Reply:
will do. we can have this in a single place